website/versioned_docs/version-0.10.0/terraform/tfctl.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Terraform Controller CLI
```

Comment:
Maybe we don't want this `tfctl` doc?

Reply:
I was hesitant to not include it, but then I thought why should we lose customer feedback.  I added a "note" saying we will be migrating it to the Weave GitOps CLI.